ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/languagenavigation/languagenavigation.html

```diff
@@ -0,0 +1,14 @@
+<a class="language-navigation-btn mini-cart-shopping" href="javascript:void(0);">
+<i class="fa fa-globe" aria-hidden="true"></i>
+  <span>EN-US</span>
```

Comment:
can we author this EN-US language instead of hard coding the language

Reply:
@Rajamouli-hcl  This is only the placeholder for template authoring. the Actual values will be pick and placed through js for lang/country combination.